### PR TITLE
order form input field validation set up

### PR DESCRIPTION
**Pull request by karenwinnielei**
Set up validation on the order form input fields and add input name values and input types to match backend data

### CODE REVIEW

File: src/components/orders/OrderFormContainer.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
   const onFormLayoutChange = ({ layout }) => {
     setFormLayout(layout);
   };
 
+  useEffect(() => {
+    forceUpdate({});
+  }, []);
+
+  const onFinish = values => {
+    console.log('Finish:', values);
```
Comment (distortedaxiom) [inline src/components/orders/OrderFormContainer.js:20]: Just a console.log

**Review by distortedaxiom — APPROVED** [review]
Forms looking great!

File: src/components/orders/OrderFormContainer.js (hunk 1 above)
Comment (karenwinnielei) [reply src/components/orders/OrderFormContainer.js:20]: Noted. Will replace with axios call after this pull request has been merged. Want to keep the buyer action and reducers consistent with the state management already built.

**Review by exploratum — APPROVED** [review]
Looks good, Like Anh mentioned don't forget to take care of the console.log when you have a chance

**Pull request merged**